pygmt/tests/test_grdview.py

```diff
@@ -34,15 +33,14 @@ def fixture_xrgrid(region):
     return grdcut(grid="@earth_relief_01d_g", region=region)
 
 
-@check_figures_equal()
-def test_grdview_grid_dataarray(gridfile, xrgrid):
+@pytest.mark.mpl_image_compare
+def test_grdview_grid_dataarray(xrgrid):
     """
     Run grdview by passing in a grid as an xarray.DataArray.
     """
-    fig_ref, fig_test = Figure(), Figure()
-    fig_ref.grdview(grid=gridfile)
-    fig_test.grdview(grid=xrgrid)
-    return fig_ref, fig_test
+    fig = Figure()
+    fig.grdview(grid=xrgrid)
```

Comment:
If you add `frame=True` to this `Figure.grdview` call and re-generate the baseline image, you will see a baseline image like:
![image](https://user-images.githubusercontent.com/3974108/113499328-1e31cf80-94e3-11eb-9372-e8b809ea8659.png)

It's obvious that the baseline image uses a Cartesian coordinate, but we expect it to be a geographic map, because xrgrid.gmt.gtype is 1.

Reply:
Running `make test`, the `result.png` image is:
![image](https://user-images.githubusercontent.com/3974108/113499522-951b9800-94e4-11eb-8bf9-b274c4d39ea9.png)

However, running `pytest pygmt/tests/test_grdview.py`, the result.png is:
![image](https://user-images.githubusercontent.com/3974108/113499534-b2506680-94e4-11eb-879f-e17c67378c40.png)

Reply:
One thing I don't get is why the `@check_figures_equal` test passes. Shouldn't the NetCDF grid be plotted on a geographic frame?